src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,41 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+    @Test
+    void testStringSplitOne() {
+        String[] result = "1".split(", ");
+        assertThat(result).contains("1");
```

Comment:
```suggestion
        assertThat("1".split(", ")).contains("1");
```
위와 같이 한 라인으로 구현하는 것은 너무 과할까?